.github/workflows/qgis-plugin-package.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7"]
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        python-version: ["3.10"]
```

Comment:
Please upgrade the https://github.com/gee-community/qgis-earthengine-plugin/blob/master/dependencies/linux/Dockerfile to 3.10 as well to keep Python versions consistent